stbt/__init__.py

```diff
@@ -64,18 +68,11 @@
     "get_frame",
     "is_screen_black",
     "load_image",
-    "match",
-    "match_all",
-    "match_text",
-    "MatchParameters",
-    "MatchResult",
-    "MatchTimeout",
```

Comment:
You probably didn't mean to remove any of these from `__all__`.

Reply:
Good spot.  Sloppy on my part.  I wonder why lint didn't object to unused imports?  Fixed